docs/source-pytorch/common/checkpointing_basic.rst

```diff
@@ -197,16 +204,31 @@ You can disable checkpointing by passing:
 
 ----
 
+
 *********************
 Resume training state
 *********************
 
 If you don't just want to load weights, but instead restore the full training, do the following:
 
+.. warning::
+
+   The parameter `resume_from_checkpoint` has been deprecated in recent versions of PyTorch Lightning.
```

Comment:
```suggestion
   The argument `resume_from_checkpoint` has been deprecated in versions of PyTorch Lightning >= 1.0.0.
```

Reply:
Also, since this will now come after the correct way, I'd word this as:

In versions of PyTorch Lightning prior to 1.0.0, resuming training state was achieved via the `resume_from_checkpoint` argument of `Trainer`. The argument has been deprecated since, in favor of the `ckpt_path` argument of `fit`.